tslint.json

```diff
@@ -65,7 +65,8 @@
     "typedef-whitespace": [true,
       ["callSignature", "noSpace"],
       ["catchClause", "noSpace"],
-      ["indexSignature", "space"]
+      ["indexSignature", "space"],
+      [ "variable-declaration", "nospace"]
```

Comment:
I'd rather just use `{}` since we use that everywhere else in the code.

Reply:
I can just remove the rule & leave it as it was.

Reply:
(agree, code already looks a certain way, so roll with it.)